source/12_policies/ppg_zh.rst

```diff
@@ -0,0 +1,129 @@
+PPG
+^^^^^^^
+
+概述
+---------
+PPG 是在 `Phasic Policy Gradient <https://arxiv.org/abs/2009.04416>`_ 中提出的。在以前的方法中，人们需要选择表示策略和价值函数是否需要分开训练还是共享全局信息。分开训练可以避免目标之间的干扰，而使用共享全局信息可以实现有用特征的共享。PPG 能够兼顾两者的优点，通过将优化分为两个阶段，一个用于推进训练，另一个用于提取特征。
+
+核心要点
+-----------
+1. PPG是一种无模型、基于策略的强化学习算法。
+
+2. PPG支持离散动作空间和连续动作空间。
+
+3. PPG支持离策略模式和在策略模式。
+
+4. PPG中有两个值网络。
+
+5. 在DI-engine的实现中，我们对离策略PPG使用了两个缓冲区，它们仅在数据使用上限（数据 "max_use" ）上有所不同。
+
+重要图示
+----------
+PPG 利用分开的策略和价值网络来减少目标之间的干扰。策略网络包括一个辅助价值头部，用于将价值的知识提取到策略网络中，具体的网络结构如下所示：
+
+.. image:: images/ppg_net.png
+   :align: center
+   :height: 250
+
+重要公式
+-------------
+PPG 的优化分为两个阶段，策略阶段和辅助阶段。在策略阶段，策略网络和价值网络的更新方式类似于 PPO。在辅助阶段，使用联合损失将价值知识提取到策略网络中：
+
+.. math::
+
+    L^{j o i n t}=L^{a u x}+\beta_{c l o n e} \cdot \hat{\mathbb{E}}_{t}\left[K L\left[\pi_{\theta_{o l d}}\left(\cdot \mid s_{t}\right), \pi_{\theta}\left(\cdot \mid s_{t}\right)\right]\right]
```

Comment:
\mathcal{L}^{\mathrm{joint}}=......
you can use latex in this way.

Reply:
okay, fixed